src/ansys/geometry/core/modeler.py

```diff
@@ -219,6 +219,16 @@ def close(self) -> None:
         """Access the client's close method."""
         return self.client.close()
 
+    def exit(self) -> None:
+        """Access the client's close method.
+
+        Notes
+        -----
+        This method is calling the same method as
+        :func:`close() <ansys.geometry.core.modeler.Modeler.close>`.
+        """
+        return self.close()
```

Comment:
Why the return? If I'm right, in `client.close()` it doesn't return anything. If the return is needed, I would document it if there is any info being returned.

https://github.com/ansys/pyansys-geometry/blob/310cde57cf4b78e056486e25206f5e15c99216cc/src/ansys/geometry/core/connection/client.py#L278-L301

Reply:
Sure, I can change that